docs/docs.css

```diff
@@ -183,6 +183,13 @@ article .eg-container {
     width: 30%;
 }
 
+#typography .text-left,
+#typography .text-center,
+#typography .text-right {
+    width: 100%;
+}
+
+
 /* media queries */
 
 @media screen and (max-width: 275px) {
```

Comment:
Why have you gone for very small and unique screen sizes?
Would be better if you used the commonly used breakpoints

Reply:
Yes I used the common breakpoints only first but as vikram suggested wherever you design is breaking you could add a media query so did that.